packages/components/src/components/ResourceDetails/ResourceDetails.js

```diff
@@ -58,7 +58,7 @@ const ResourceDetails = ({
   return (
     <div className="tkn--resourcedetails">
       <h1>{resource.metadata.name}</h1>
-      <Tabs selected={0}>
+      <Tabs selected={0} aria-label="Resource details">
```

Comment:
Should `Resource details` be translated ?

Reply:
yeah no harm adding it, I'll do that now. Technically not required but good to be consistent.